modules/distr-tracing-otel-troubleshoot-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-troubleshooting.adoc
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-troubleshoot-logs_{context}"]
+= Getting the OpenTelemetry Collector logs
+
+You can get the logs for the OpenTelemetry Collector as follows.
```

Comment:
I think this introduction could be a little more specific. Maybe something like, "You can specify your desired log level for the OpenTelemetry Collector as shown in the following example..."